tests/introspection/tests.py

```diff
@@ -171,8 +171,8 @@ def test_get_indexes(self):
         with connection.cursor() as cursor:
             indexes = connection.introspection.get_indexes(cursor, Article._meta.db_table)
         if connection.features.can_introspect_index_type:
-            self.assertIn('type', indexes['reporter_id'])
-            indexes['reporter_id'].pop('type')
+            index_type = indexes['reporter_id'].pop('type', None)
+            self.assertEqual(index_type, 'btree')
```

Comment:
Anyway, I'd add `self.assertIsNotNone(index_type)` and move the 'btree' check under a postgresql vendor check for now so as not to block the rest of the PR.